pkg/handler/handler.go

```diff
@@ -364,8 +364,16 @@ func (h handler) update(w http.ResponseWriter, r *http.Request, params map[strin
 			writeResponse(w, http.StatusBadRequest, broker.ErrorResponse{Description: err.Error()})
 		case broker.ErrorNoUpdateRequested:
 			writeResponse(w, http.StatusOK, resp)
-		default:
+		case broker.ErrorNoUpdateRequested:
+			writeResponse(w, http.StatusOK, resp)
+		case broker.ErrorPlanNotFound:
+			writeResponse(w, http.StatusBadRequest, broker.ErrorResponse{Description: err.Error()})
+		case broker.ErrorParameterNotUpdatable:
 			writeResponse(w, http.StatusBadRequest, broker.ErrorResponse{Description: err.Error()})
+		case broker.ErrorPlanUpdateNotPossible:
+			writeResponse(w, http.StatusBadRequest, broker.ErrorResponse{Description: err.Error()})
```

Comment:
I think you can do the multiple case by comma separating them:
```
case broker.ErrorPlanNotFound, broker.ErrorParameterNotUpdatable, broker.ErrorPlanUpdateNotPossible:
	writeResponse(w, http.StatusBadRequest, broker.ErrorResponse{Description: err.Error()})
default:
```

Reply:
Brilliant! I rescind my earlier complaints.